.gitlab-ci.yml

```diff
@@ -7,19 +7,19 @@ stages:
   - CHAOS
```

Comment:
@harshita-sharma-mayadata for chaos stage did we included any jobs. If it is not there please comment on this stage from gitlab.